src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -61,4 +71,71 @@ class WorkspaceService(protected val argUserInfo: UserInfo, val rawlsDAO: RawlsD
     }
   }
 
+  def exportWorkspaceAttributes(workspaceNamespace: String, workspaceName: String, filename: String): Future[PerRequestMessage] = {
+    Try(rawlsDAO.getWorkspace(workspaceNamespace, workspaceName)) match {
+      case Failure(regret) => Future(RequestCompleteWithErrorReport(StatusCodes.BadRequest, regret.getMessage))
+      case Success(workspaceFuture) => workspaceFuture map { workspaceResponse =>
+          val headerString = "workspace:" + (workspaceResponse.workspace.get.attributes map { attribute =>
+                attribute._1.name}).mkString("\t").replaceAll("description\t", "")
+          val valueString = (workspaceResponse.workspace.get.attributes map { attribute =>
+              impAttributeFormat.write(attribute._2).toString().replaceAll("\"","")}).mkString("\t").replaceAll("null\t", "")
+          RequestComplete(StatusCodes.OK, headerString + "\n" + valueString)
+        }
+    }
+  }
+
+  def importAttributesFromTSV(workspaceNamespace: String, workspaceName: String, tsvString: String): Future[PerRequestMessage] = {
+    Try(TSVParser.parse(tsvString)) match {
+      case Failure(regret) => Future(RequestCompleteWithErrorReport(StatusCodes.BadRequest, regret.getMessage))
+      case Success(tsv) =>
+          tsv.firstColumnHeader.split(":")(0)  match {
+            case "workspace" =>
+                importWorkspaceAttributeTSV(workspaceNamespace, workspaceName, tsv)
+            case _ =>
+                Future(RequestCompleteWithErrorReport(StatusCodes.BadRequest, "Invalid first column header should start with \"workspace\""))
+        }
+    }
+  }
+
+   /*
+  Import attributes on a workspace from a TSV file
+  */
+  private def importWorkspaceAttributeTSV(workspaceNamespace: String, workspaceName: String, tsv: TSVLoadFile): Future[PerRequestMessage] = {
+    checkFirstRowDistinct(tsv) {
+        val attributePairs = colNamesToWorkspaceAttributeNames(tsv.headers).zip(tsv.tsvData.head)
+        Try(scala.util.parsing.json.JSONObject(attributePairs.toMap).toString().parseJson.asJsObject.convertTo[AttributeMap]) match {
+          case Failure(ex:ParsingException) => Future(RequestCompleteWithErrorReport(StatusCodes.BadRequest, "Invalid json supplied", ex))
```

Comment:
if this is the error that the user is going to see, "Invalid json supplied" will not make a whole lot of sense to them because they're uploading a tsv